src/test/java/uk/gov/hmcts/reform/sscs/services/CaseLoaderServiceTest.java

```diff
@@ -181,4 +182,21 @@ public void shouldThrowExceptionGivenNoReferenceFileLoaded() {
         caseLoaderService.process();
     }
 
+    @Test(expected = CcdException.class)
+    public void shouldThrowExceptionWhileUpdatingCase() {
+        final SscsCaseDetails sscsCaseDetails = SscsCaseDetails.builder().build();
+        caseData.setCaseReference(null);
+        caseData.setCcdCaseId("1234567890");
+
+        when(transformService.transform(inputStream)).thenReturn(newArrayList(caseData));
+
+        when(searchCcdCaseService.findCaseByCaseRefOrCaseId(eq(caseData), eq(idamTokens)))
+                .thenReturn(sscsCaseDetails);
+
+        doThrow(RuntimeException.class).when(ccdCasesSender).sendUpdateCcdCases(any(), any(), any());
+
+        caseLoaderService.process();
```

Comment:
I suppose it would be difficult to verify that the file was also moved once the error is thrown.